sdk/eventhub/event-hubs/src/models/public.ts

```diff
@@ -111,18 +111,22 @@ export enum CloseReason {
  */
 export interface EventHubClientOptions {
   /**
-   * @property
+   * A custom endpoint to use when connection to the Event Hubs service.
+   * This can be useful when your network does not allow connecting to the
+   * standard Azure Event Hubs endpoint address, but does allow connecting
+   * through an intermediary.
```

Comment:
Is there an example we can show here like we do for the fully qualified namespace?

Reply:
Sure, I added an example.